uiowa/uids
# Feature: Added banner pre-title

# How to test

- Go to https://uids.brand.uiowa.edu/branches/feature_banner_pre_title/components/detail/banner--default.html and click through all the banners and verify that a pre-title field is appearing above the banner title field in the same headline style as the banner title. 
  - The "Highlight" styles will render without the highlight background
- Review banner template changes
- Review `banner.scss` changes
